### PR TITLE
cv-galactic-tech-innovations.html

### PR DESCRIPTION
**Pull request by JuanBerrocal**
cv_GalacticTech_Innovations.html

### CODE REVIEW

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 1]
```diff
@@ -0,0 +1,692 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html:8]: You need to add Raleway font 

`<link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">`

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 2]
```diff
@@ -0,0 +1,692 @@
...
+        --gray_color_900: #5c5a58;
+      }
+      /* General styles */
+      * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+      html {
+        font-family: Roboto, sans-serif;
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: Use Raleway font by default

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 3]
```diff
@@ -0,0 +1,692 @@
...
+        text-transform: uppercase;
+        color: var(--primary_color);
+        padding: 0 var(--spacing_16) 0 0;
+      }
+      .header-left-gap {
+        display: none;
+      }
+      .header-right-gap {
+        display: block;
+        /*min-width: fit-content;*/
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: remove comment

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 4]
```diff
@@ -0,0 +1,692 @@
...
+        /*font-family: 'Raleway';*/
+        color: var(--primary_color);
+        font-size: 20px;
+        font-weight: 540;
+        letter-spacing: 0.96px;
+        text-transform: uppercase;
+      }
+      .aside-section-list {
+        gap: var(--spacing_16);
+        /*font-family: 'Raleway';*/
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: remove comment

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 5]
```diff
@@ -0,0 +1,692 @@
...
+        display: flex;
+        flex-direction: column;
+        font-family: Roboto, sans-serif;
+        padding: 0 var(--spacing_16) 0 var(--spacing_16);
+        gap: var(--spacing_16);
+        border-left: 1px solid var(--primary_color);
+        min-height: 100%;
+      }
+      .aside-section-name {
+        /*font-family: 'Raleway';*/
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: remove comment

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 6]
```diff
@@ -0,0 +1,692 @@
...
+      .aside-area {
+        display: flex;
+        flex-direction: column;
+        text-align: left;
+        gap: var(--spacing_64);
+      }
+      .aside-section {
+        display: flex;
+        flex-direction: column;
+        font-family: Roboto, sans-serif;
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: Remove  it, we use by default Raleway font

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 7]
```diff
@@ -0,0 +1,692 @@
...
+        gap: var(--spacing_4);
+        align-self: stretch;
+      }
+      .header-title {
+        min-width: 100%;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        flex: auto;
+        /*justify-content: space-between;*/
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: remove comment

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 8]
```diff
@@ -0,0 +1,692 @@
...
+      }
+      .header-name {
+        display: inline-block;
+        /*min-width: fit-content;*/
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: remove comment

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 9]
```diff
@@ -0,0 +1,692 @@
...
+        font-size: 36px;
+        font-weight: 500;
+        letter-spacing: 2.4px;
+        text-transform: uppercase;
+        color: var(--gray_color_900);
+        white-space: nowrap;
+        padding-right: var(--spacing_16);
+      }
+      .header-job-position {
+        font-family: 'Raleway';
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: We don't need it. Raleway font is by default

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 10]
```diff
@@ -0,0 +1,692 @@
...
+        padding: 0 var(--spacing_16) 0 var(--spacing_16);
+        gap: var(--spacing_16);
+        border-left: 1px solid var(--primary_color);
+        min-height: 100%;
+      }
+      .aside-section-name {
+        /*font-family: 'Raleway';*/
+        color: var(--primary_color);
+        font-size: 20px;
+        font-weight: 540;
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: replace by font-weight: 600

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 11]
```diff
@@ -0,0 +1,692 @@
...
+        letter-spacing: 0.96px;
+        text-transform: uppercase;
+      }
+      .aside-section-list {
+        gap: var(--spacing_16);
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html:187]: gap is a flex or grid property. Add display: flex and flex-direction: column here

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 12]
```diff
@@ -0,0 +1,692 @@
...
+
+      .main-chapter-header {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        min-width: fit-content;
+      }
+      .main-chapter-header-title {
+        font-size: 24px;
+        font-weight: 580;
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: Replace by 600.
580 is not a valid weight value, the values are 100, 200, 300, 400, .... 900.

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 13]
```diff
@@ -0,0 +1,692 @@
...
+        color: var(--primary_color);
+        text-transform: uppercase;
+        padding: 0 var(--spacing_32) 0 0;
+        white-space: nowrap;
+        letter-spacing: 1px;
+        min-width: 25%;
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: Remove min-width

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 14]
```diff
@@ -0,0 +1,692 @@
...
+        <div class="main-chapter-gap"></div>
+
+        <div class="main-chapter">
+          <div class="main-chapter-header">
+            <div class="main-chapter-header-title">educación</div>
+            <div class="main-chapter-header-line"></div>
+          </div>
+
+          <div class="main-chapter-body">
+            <div class="main-chapter-article">
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html:618]: Review the education sections and update acording to design.

File: export-samples/html/cv-galactic-tech-innovations.html [hunk 15]
```diff
@@ -0,0 +1,692 @@
...
+      }
+
+      .main-chapter-article-subtitle {
+        font-size: 16px;
+        font-weight: 600;
+        color: var(--black_color);
+      }
+
+      .main-chapter-article-body {
+        color: var(--gray_color_800);
```
Comment (juanpms2) [inline export-samples/html/cv-galactic-tech-innovations.html]: the color is --gray_color_900

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 1 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html:8]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 2 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 3 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 4 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 5 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 6 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 7 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 8 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 9 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 10 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 11 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html:187]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 12 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 13 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 14 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html:618]: OK

File: export-samples/html/cv-galactic-tech-innovations.html (hunk 15 above)
Comment (JuanBerrocal) [reply export-samples/html/cv-galactic-tech-innovations.html]: OK

**Comment by JuanBerrocal** [comment]
OK. Changes for 05.03.2024 revision made.

**Pull request merged**